config.yml

```diff
@@ -1,29 +1,27 @@
-version: '9'
+version: '10'
 name: Android Open Source Project
 type: release
 variant: user
-datetime: 1555835843
+datetime: 1567667515
 host: android
 user: build
 platform:
-  ref: android-9.0.0_r45
+  ref: android-10.0.0_r2
   extra_remotes:
   - name: github
     fetch: https://github.com
   extra_projects:
-  - name: anestisb/android-prepare-vendor
+  - name: RattlesnakeOS/android-prepare-vendor
```

Comment:
Why RattlesnakeOS?

Reply:
their fork is the only one with 10 support atm.